modules/core/04-channel/keeper/handshake.go

```diff
@@ -41,6 +42,15 @@ func (k Keeper) ChanOpenInit(
 	// corresponding to the connection to chain Z for the logic to be meaningful.
 	// ******************************************************************************************
 
+	// getVersions := connectionEnd.GetVersions()
+	// if len(getVersions) != 1 {
+	// 	return "", nil, errorsmod.Wrapf(
+	// 		connectiontypes.ErrInvalidVersion,
+	// 		"single version must be negotiated on connection before opening channel, got: %v",
+	// 		getVersions,
+	// 	)
+	// }
+
 	getVersions := connectionEnd.GetVersions()
```

Comment:
I think its fine to do this check in both single hop and multihop case. THe connection handshake should ensure for us that the two endpoints agree on the version. So we should be able to just check our own endpoint version here.